### PR TITLE
featrue: Include go-ds-pebble as built-in plugin

### PR DESCRIPTION
**Pull request by guojidan**
closed: #10347 

I have tested it locally, It seems to be working properly 🤔

### CODE REVIEW

File: docs/datastores.md [hunk 1]
```diff
@@ -51,6 +51,17 @@ Uses [badger](https://github.com/dgraph-io/badger) as a key value store.
...
+## pebbleds
+
+Uses [pebble](https://github.com/cockroachdb/pebble) as a key value store
+
+```json
+{
+	"type": "pebbleds",
+	"path": "<location of pebble inside repo>"
+}
+```
```
Comment (lidel) [inline docs/datastores.md:63]: @hsanjuan I know pebble is used by default in IPFS Cluster, and there is a lot of custom configuration in https://github.com/ipfs-cluster/ipfs-cluster/blob/v1.0.8/datastore/pebble/config.go#L25-L70

Two questions:
1. Does it make sense to expose them all via config here (as optional, use only when set)
2. For implicit defaults, should we use defaults from pebble, or ones from ipfs-cluster?

**Review by lidel — CHANGES_REQUESTED** [review]
Thank you for taking a stab at this @guojidan.
Are you able to add tests in similar to ones we have for `badgerds` in

- test/sharness/t0060-daemon.sh
- test/sharness/t0025-datastores.sh 

?

**Comment by guojidan** [comment]
I will do this in the next few days

**Comment by lidel** [comment]
Triage question: @guojidan will you have interest and time to work on this, or is it ok for someone else to pick up the pebble work?

**Comment by lidel** [comment]
@guojidan no problem!  @gammazero will pick this up after we are done with more pressing stuff. 

closing this PR, we will continue in #10347 + new one.

**Pull request closed without merging**